packages/neuron-wallet/src/services/tx/transaction-service.ts

```diff
@@ -66,23 +66,38 @@ export class TransactionsService {
       .map(s => assetAccountInfo.generateAnyoneCanPayScript(s.args).computeHash())
     const allLockHashes: string[] = lockHashes.concat(anyoneCanPayLockHashes)
 
+    const connection = getConnection()
+    const repository = connection.getRepository(TransactionEntity)
+
+    let allTxHashes: string[] = []
+
     if (type === SearchType.Address) {
-      const hash = AddressParser.parse(searchValue).computeHash()
-      if (lockHashes.includes(hash)) {
-        lockHashes = [hash]
+      const lockHashToSearch = AddressParser.parse(searchValue).computeHash()
+      if (lockHashes.includes(lockHashToSearch)) {
+        lockHashes = [lockHashToSearch]
+        allTxHashes = await repository.createQueryBuilder('tx').select('tx.hash', 'txHash').where(
+          `tx.hash
+          IN
+            (
+              SELECT output.transactionHash FROM output WHERE output.lockHash in (:...lockHashes)
+              UNION
+              SELECT input.transactionHash FROM input WHERE input.lockHash in (:...lockHashes)
+            )
+          `,
+          { lockHashes }
+        )
+          .orderBy('tx.timestamp', 'DESC')
+          .getRawMany().then(txs => txs.map(tx => tx.txHash))
+        console.log({
```

Comment:
I guess we won't need this log to be merged.

Reply:
😢 The eslint didn't hold it. I'll fix it.

Reply:
Updated in https://github.com/nervosnetwork/neuron/pull/1733/commits/1c3156d84c97dae96e17ab871b27a5bc15e3794c